scripts/serverSetup.md

```diff
@@ -1,49 +0,0 @@
-# Getting started with YesWorkflow Web Components
```

Comment:
is this information located somewhere else?

Reply:
Yea most of this is redundant with the README